docs/user_manual.md

```diff
@@ -878,6 +879,157 @@ datum stats -p test_project
 
 </details>
 
+
+### Validate project annotations
+
+This command inspects annotations based on the task type
+and stores the result in JSON file.
+
+The task types supported are `classification`, `detection`, and `segmentation`.
+
+The validation result contains
+- annotation statistics based on the task type
+- validation reports, such as
+    - items not having annotations
+    - items having undefined annotations
+    - imbalanced distribution in class/attributes
+    - too small or large values (items_far_from_mean)
+- summary
+
+Usage:
+
+``` bash
+datum validate --help
+
+datum validate -p <project dir> <task_type>
+```
+
+Validation Result:
+
+<details>
+
+``` bash
+{
+    'statistics': {
+        ## common statistics
+        'label_distribution': {
+            'defined_labels': <dict>,   # <label:str>: <count:int>
+            'undefined_labels': <dict>
+            #  <label:str>: {
+            #    'count': <int>,
+            #    'items_with_undefined_label': [(<item.id>, <item.subset>), ]
+            #  }
+        },
+        'attribute_distribution': {
+            'defined_attributes': <dict>,
+            #  <label:str>: {
+            #    <attribute:str>: {
+            #      'distribution': {<attr_value:str>: <count:int>, },
+            #      'items_missing_attribute': [(<item.id>, <item.subset>), ]
+            #    }
+            #  }
+            'undefined_attributes': <dict>
+            #  <label:str>: {
+            #    <attribute:str>: {
+            #      'distribution': {<attr_value:str>: <count:int>, },
+            #      'items_with_undefined_attr': [(<item.id>, <item.subset>), ]
+            #    }
+            #  }
+        },
+        'total_ann_count': <int>,
+        'items_missing_annotation': <list>, # [(<item.id>, <item.subset>), ]
+
+        ## statistics for classification task
+        'items_with_multiple_labels': <list>, # [(<item.id>, <item.subset>), ]
+
+        ## statistics for detection task
+        'items_with_invalid_value': <dict>,
+        #  '(<item.id>, <item.subset>)' : {
+        #    <ann.id> : [ <property:str>, ]
+        #  }
+        #  properties: 'x', 'y', 'width', 'height',
+        #              'area(wxh)', 'ratio(w/h)', 'short', 'long'
```

Comment:
Clarify the meaning of `short` and `long`

Reply:
OK. I will.